plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -735,7 +734,7 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
           }
 
           // disable rotate gesture when scale is detected first
```

Comment:
Not related to this pr - but wonder whether it's a correct behaviour, disable rotate when scale happens first, if scale/rotate are both allowed with `GestureSettings.simultaneousRotateAndPinchToZoomEnabled`

Reply:
IMHO worth adding an issue to check @paulrenn67